### ISSUE TITLE
Update accessibility conformance report and accessibility statement

### ISSUE THREAD

**Issue by MI6-94** [mi6/ic-design-system#678]
## Summary

12 months has passed since we last produced an accessibility conformance report. This document should be updated along with an accessibility statement.

Internal ticket **2455** is set up for tracking progress on a draft copy of the Conformance Report.

## 💬 Description

- Test the full website via usual methods and update the accessibility conformance report to reflect any findings.

We're going to break down the work into the four overall criteria. Each must be assessed up to AAA level, as the ICDS is an exemplar.
- [ ] Percievable criteria (1.x.x)
- [ ] Operable criteria (2.x.x)
- [ ] Understandable criteria (3.x.x)
- [ ] Robust criteria (4.x.x)

### Expected code changes
- [ ] Update the Accessibility Statement page in the design-system codebase, based on the results from the Conformance Report.
- [ ] Replace the Conformance Report document (ensuring that it's properly stripped of exif data)

## Additional info

- Use our guidance to know how to produce an accessibility conformance report and accessibility statement. https://design.sis.gov.uk/accessibility/documenting/when-to-create-and-update

**Comment by GCHQ-Developer-299** [comment]
Ignore my previous mention, this is actually related to [internal issue 2455](https://github.com/mi6/ic-ui-kit/issues/1281)

**Comment by GCHQ-Developer-299** [comment]
WCAG 2.2 Criteria have been added to the draft ACR document and i've started populating them. Getting us ahead of the curve :)
